datasets/wino_bias/wino_bias.py

```diff
@@ -68,12 +71,56 @@ class WinoBias(datasets.GeneratorBasedBuilder):
     # You will be able to load one or the other configurations in the following list with
     # data = datasets.load_dataset('my_dataset', 'first_domain')
     # data = datasets.load_dataset('my_dataset', 'second_domain')
+    def __init__(self, *args, writer_batch_size=None, **kwargs):
+        super(WinoBias, self).__init__(*args, **kwargs)
+        # Batch size used by the ArrowWriter
+        # It defines the number of samples that are kept in memory before writing them
+        # and also the length of the arrow chunks
+        # None means that the ArrowWriter will use its default value
+        self._writer_batch_size = writer_batch_size or 100
+
     BUILDER_CONFIGS = [
-        datasets.BuilderConfig(
-            name="wino_bias",
-            version=VERSION,
-            description="WinoBias: Winograd-schema dataset for detecting gender bias",
+        WinoBiasConfig(
+            name='dev_type1_pro',
+            description = "winoBias dev_type1_pro_stereotype data in cornll format",
+            data_url = _URL + "/dev_type1_pro_stereotype.v4_auto_conll"
+        ),
```

Comment:
I think we should only have the configurations "type1_pro", "type1_anti", "type2_pro" and "type2_anti".
Then each configuration can have train/dev/test splits. What do you think ?

Reply:
Good point! Will update that. Thanks!